pkg/repo2cpe/mapping.go

```diff
@@ -50,7 +50,11 @@ func NewMapping() *Mapping {
 
 // Load loads the contents of the RHELv2CPERepoName file in the given directory into the Mapping.
 func (m *Mapping) Load(dir string) error {
-	path := filepath.Join(dir, RHELv2CPERepoName)
+	return m.LoadFile(filepath.Join(dir, RHELv2CPERepoName))
+}
+
+// LoadFile loads the contents of the specified file into the Mapping.
+func (m *Mapping) LoadFile(path string) error {
```

Comment:
Why not just use the `Load` function?

Reply:
`Load() assumes the file's path within the directory. `LoadFile()` doesn't have that assumption and frees callers from having the setup of the file in a specific way, decoupling them.